EmojicodeCompiler/Lexer.cpp

```diff
@@ -142,6 +143,29 @@ TokenStream lex(const char *path) {
                         /* End of variable */
                         nextToken = true;
                     }
+					else if (c == 0xFE0F) {
+						// Variation Selector 16
+						possiblyIdentifier = true;
+						continue;
+					}
+					else if (possiblyIdentifier) {
+						if (c == 0x20E3) {
+							// COMBINING ENCLOSING KEYCAP: This is a keycap emoji
+							auto first = token->value[0];
+							token->value.clear();
+							token->value.push_back(first);
+							token->type_ = TokenType::Identifier;
+							nextToken = true;
+							continue;
+						}
+						else {
+							// There was just a misplaced Variation Selector, delete and move on
+							possiblyIdentifier = false;
+							token->value.pop_back();
```

Comment:
I don’t get what this `pop_back` call here is good for. You haven't pushed anything you would want to pop, have you?